webviz_subsurface/_datainput/opm_init_io.py

```diff
@@ -116,25 +116,27 @@ def get_values(self) -> List[VariateAndValues]:
 
     # pylint: disable=R0201
     def formation_volume_factor(
-        self, gas_oil_ratio: List[float], pressure: List[float]
+        self, set_value: List[float], pressure: List[float]
     ) -> List[float]:
         """
         Does only return an empty list for now.
         """
-        if len(gas_oil_ratio) != len(pressure):
+        if len(set_value) != len(pressure):
             raise ValueError("rs and po arguments must be of same size")
```

Comment:
Suggest to generalize this now
```suggestion
            raise ValueError("Rs / Rv and pressure arguments must be of same size")
```